src/main/resources/assets/minecolonies/models/item/super_bread.json

```diff
@@ -1,6 +0,0 @@
-{
-	"parent": "item/generated",
-	"textures": {
```

Comment:
@Mekle001 is this one unused?

Reply:
Currently, yes. That can rapidly change, if somebody has an idea for effect.

Reply:
that would probably fit moving to unused resources